src/components/Pagination/index.tsx

```diff
@@ -17,6 +17,10 @@ export default function Pagination({ currentPage, totalPages, onPageChange }: Pa
     onPageChange(currentPage - 1);
   };
 
+  const handleNextClick = () => {
+    onPageChange(currentPage + 1);
```

Comment:
Use callback syntax as state change relies in previous state `onPageChange(cur=>cur+1)`